plugins/techdocs/src/home/components/TechDocsCustomHome.tsx

```diff
@@ -119,15 +147,21 @@ const CustomPanel = ({
 
```

Comment:
A question for maintainers:
We have another use case where we would like to filter documentation sections based on a config file (in order to prevent teams at the company just registering their docs in any section in their yaml files). I was thinking to pass the config along with the entity for optional use in line 144 above like so:
```
const configApi: ConfigApi = useApi(configApiRef);
return (
      typeof config.filterPredicate === 'function' &&
      config.filterPredicate(entity, configApi)
    );
```
But I wasn't sure if that was an anit-pattern or if there is a better recommended way. Any tips?

Reply:
At our company, we allow teams to manage & register their own catalog-info.yaml files. So they can technically add whatever they want to their yaml files.
We want a way to control our docs page sections (potentially through a config file), because if we filter on tags or another field in the yaml file - any team can add the required filter to their yaml file. 
So say our filterPredicate is:
```
filterPredicate: entity =>
          entity?.metadata?.tags?.includes('golden-path') ?? false
```
any team can add this to their yaml file:
```
metadata:
  tags: 
    - 'golden-path'
```
and it would show up in our docs section for anyone to view.